pages/common/jello.md

```diff
@@ -0,0 +1,32 @@
+# jello
+
+> A command-line JSON processor using python syntax.
+> More information: <https://github.com/kellyjonbrazil/jello>.
+
+- Output JSON or JSON Lines STDIN data, in pretty-print format:
```

Comment:
```suggestion
- Output JSON or JSON-Lines STDIN data, in pretty-print format:
```
`JSON-Lines` is a file format right? If yes I think my suggestion makes it a bit readable. Also I think piping is not the same as reading from STDIN. Maybe we could even rewrite it like that:
```suggestion
- Pretty-print JSON or JSON-Lines input to stdout:
```
The same would apply to all other examples. But maybe let's wait for the other's opinions.

Reply:
Maybe "Pretty-print JSON or JSON-Lines data from stdin to stdout"?

Reply:
Sounds good to me!
```suggestion
- Pretty-print JSON or JSON-Lines data from stdin to stdout:
```